examples/export_figure.py

```diff
@@ -99,6 +99,10 @@
 scaled_export_figure = viewer.export_figure(scale_factor=5)
 viewer.theme = "dark"
 
+# Add scale bar of a defined length to the exported figure
+viewer.scale_bar.visible = True
+viewer.scale_bar.length = 250
+
```

Comment:
Actually, this doesn't work currently, it should be before the `viewer.export_figure()` line :sweat_smile: